src/helpers/assets.js

```diff
@@ -205,7 +205,7 @@ export const buildUniqueTokenList = (uniqueTokens, selectedShowcaseTokens) => {
       }
     }
     let tokens = compact(tokensRow);
-    tokens = chunk(tokens, tokens.length > 25 ? 4 : 25);
+    tokens = chunk(tokens, 50); // cap NFT cards per family at 100
```

Comment:
comment says 100 and parameter says 50 🤔

Reply:
Each `tokenRow` is two token cards, so chunking at 50 tokenRows means 100 cards 💯